nexus/src/external_api/http_entrypoints.rs

```diff
@@ -5075,6 +5083,197 @@ async fn updates_refresh(
     apictx.external_latencies.instrument_dropshot_handler(&rqctx, handler).await
 }
 
+/// View system version and update status
+#[endpoint {
+     method = GET,
+     path = "/v1/system/update/version",
+     tags = ["system"],
+}]
+async fn system_version(
+    rqctx: Arc<RequestContext<Arc<ServerContext>>>,
+) -> Result<HttpResponseOk<views::SystemVersion>, HttpError> {
+    let apictx = rqctx.context();
+    let _nexus = &apictx.nexus;
```

Comment:
Pretty sure this is already on your radar, but we should consider a non-stub call before merging